src/entity/CommonListing.java

```diff
@@ -29,6 +29,7 @@ public String getFilter() {
         return filter;
     }
 
+    @Override
```

Comment:
This method is not useful anymore, as the implementation of the placeInfo use case was completely reformulated